### PR TITLE
Improve lottery plugin

### PR DESCRIPTION
**Pull request by Bryan-0**
- Remove all unnecessary ``this.errorReply`` from ``this.can`` function.

### CODE REVIEW

File: chat-plugins/lottery.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ exports.commands = {
                 this.privateModCommand("(" + Tools.escapeHTML(user.name) + " has ended the game of lottery.)");
                 break;
             case 'setlimit':
-                if (!this.can('hotpatch')) return this.errorReply("Access denied.");
+                if (!this.can('hotpatch', null, room)) return false;
```
Comment (panpawn) [inline chat-plugins/lottery.js]: ``hotpatch`` is not a room command, I think ``if (!this.can('hotpatch')) return false;`` would be good here.

File: chat-plugins/lottery.js [hunk 2]
```diff
@@ -144,7 +144,7 @@ exports.commands = {
                 return this.sendReplyBox("The current lottery pot is worth: <b>" + room.lottery.pot + "</b> bucks.");
                 break;
             case 'obj':
-                if (!this.can('hotpatch')) return this.errorReply("Access denied.");
+                if (!this.can('hotpatch', null, room)) return false;
```
Comment (panpawn) [inline chat-plugins/lottery.js]: Here too.

**Comment by Bryan-0** [comment]
Done @panpawn.

**Comment by panpawn** [comment]
Thanks

**Pull request merged**